.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,11 @@
 
 **What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
 
+**What is your application server (e.g. `puma`, `unicorn`, `thin`, `webrick`,
+etc.)?**
+
+**Is your application server multi-threaded or multi-process?**
```

Comment:
Great explanation. Does it appear elsewhere? Can one of these questions link to that explanation?